euroe-demo/.prettierrc

```
@@ -0,0 +1,7 @@
+{
+  "trailingComma": "all",
+  "semi": true,
+  "singleQuote": true,
+  "printWidth": 120,
+  "bracketSpacing": true
+}
```

This is the usual config we use for formatting with prettier.

```suggestion
{
  "trailingComma": "all",
  "tabWidth": 4,
  "semi": true,
  "singleQuote": true,
  "printWidth": 120,
  "bracketSpacing": true
}
```

---

Hmm, I'm pretty sure I copied it from the dapp-template.

```
{
    "trailingComma": "all",
    "semi": true,
    "singleQuote": true,
    "printWidth": 120,
    "bracketSpacing": true
}
```

I'll add the tabwidth there to then.

---

Yes, sorry that'll be my mistake then 🙈